conf/foreign.routes

```diff
@@ -152,3 +152,17 @@ POST       /:taxYear/foreign-property/expenses/:countryCode/check-your-answers
 
 GET        /:taxYear/foreign-property/expenses/:countryCode/complete-yes-no                                   controllers.foreign.expenses.ForeignExpensesSectionCompleteController.onPageLoad(taxYear: Int, countryCode: String)
 POST       /:taxYear/foreign-property/expenses/:countryCode/complete-yes-no                                   controllers.foreign.expenses.ForeignExpensesSectionCompleteController.onSubmit(taxYear: Int, countryCode: String)
+
```

Comment:
@wp94 Bette to create a separate Roued for UK and foreign properties 
Refer to https://github.com/hmrc/income-tax-property-frontend/pull/473,